### PR TITLE
Create Java code snippets for sets redis#3626

### PR DESCRIPTION
**Pull request by suraj11chauhan**
I have added the Java code example for the 'set' operation in the 'io.redis.examples' package. Please review my code! set_tutorial#3626 @sav-norem

### CODE REVIEW

File: src/test/java/io/redis/examples/SetsExample.java [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+        jedis.sadd("bikes:racing:usa", "bike:1", "bike:4");
+
+        Set<String> res13 = jedis.sdiff("bikes:racing:france", "bikes:racing:usa");
+        System.out.println(res13);  // >>> [bike:2, bike:3]
+
+        // REMOVE_START
+        assertEquals("[bike:2, bike:3]",res13.toString());
+        jedis.del("bikes:racing:france");
+        jedis.del("bikes:racing:usa");
+        // REMOVE_END
```
Comment (dwdougherty) [inline src/test/java/io/redis/examples/SetsExample.java:123]:
```suggestion
        // REMOVE_END
        // STEP_END
```

File: src/test/java/io/redis/examples/SetsExample.java [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        // REMOVE_START
+        assertEquals("[bike:1]",res14.toString());
+        assertEquals("[bike:1, bike:2, bike:3, bike:4]",res15.toString());
+        assertEquals("[]",res16.toString());
+        assertEquals("[bike:4]",res17.toString());
+        assertEquals("[bike:2, bike:3]",res18.toString());
+        jedis.del("bikes:racing:france");
+        jedis.del("bikes:racing:usa");
+        jedis.del("bikes:racing:italy");
+        // REMOVE_END
```
Comment (dwdougherty) [inline src/test/java/io/redis/examples/SetsExample.java:154]:
```suggestion
        // REMOVE_END
        // STEP_END
```

**Review by dwdougherty — CHANGES_REQUESTED** [review]
Thanks for re-submitting, Suraj.

The code is missing a couple of STEP_END lines, which breaks the doc build. Please correct per the comments below.

**Comment by suraj11chauhan** [comment]
I have added '// STEP_END' lines to the code. Thank you!! @dwdougherty

**Review by dwdougherty — APPROVED** [review]
LGTM. Docs build correctly.

**Pull request merged**